### PR TITLE
Add FieldKit to CDN JS

### PR DESCRIPTION
**Pull request by iamJoeTaylor**
Please let me know if I'm doing this incorrectly.

Thanks,
-Joe

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Hi @iamJoeTaylor, would you please squash them into only one commit and use much meaningful commit message, thanks.

cc @cdnjs/intern please also help review this PR.

File: ajax/libs/field-kit/package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "npmName": "field-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
+      ]
+    }
+  ],
+  "repository": "square/field-kit"
```
Comment (PeterDaveHello) [inline ajax/libs/field-kit/package.json]: This is not valid for us, should use a object to say its type and url.

File: ajax/libs/field-kit/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "field-kit",
+  "filename": "field-kit.min.js",
+  "version": "2.0.4",
+  "description": "Utilities to effectively manage text entry on the web.",
+  "homepage": "http://square.github.io/field-kit/",
```
Comment (PeterDaveHello) [inline ajax/libs/field-kit/package.json]: I'll respectfully suggest you to use https here.

File: ajax/libs/field-kit/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "format"
+  ],
+  "author": "Square, Inc.",
+  "license": "Apache-2.0",
+  "npmName": "field-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.js"
```
Comment (LeaYeh) [inline ajax/libs/field-kit/package.json]: "*.+(js|map)"
@iamJoeTaylor i think we also need `field-kit.min.js.map` and `field-kit.js.map`

**Comment by iamJoeTaylor** [comment]
Thanks for all the feedback. I think I've addressed all the issues.

File: ajax/libs/field-kit/package.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "format"
+  ],
+  "author": "Square, Inc.",
+  "license": "Apache-2.0",
+  "npmName": "field-kit",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "*.+(js|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/field-kit/package.json]: I think use "**/*" will be even better, what do you think?

**Comment by PeterDaveHello** [comment]
And it'll be even better if you can change the commit message from "add field-kit lib" to "add field-kit lib v2.0.4", thanks!

**Comment by iamJoeTaylor** [comment]
@PeterDaveHello done

**Comment by PeterDaveHello** [comment]
Thanks @iamJoeTaylor

**Pull request merged**